src/map-marker/__tests__/floating-marker.scenario.js

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+import {Select} from '../../select/index.js';
+import {FloatingMarker} from '../index.js';
+import {
+  FLOATING_MARKER_ANCHOR_POSITIONS,
+  FLOATING_MARKER_SIZES,
+} from '../constants.js';
+import TileGrid from './tile-grid.js';
+import {Checkbox, LABEL_PLACEMENT} from '../../checkbox/index.js';
+import {Input} from '../../input/index.js';
+import Upload from '../../icon/upload.js';
+import Search from '../../icon/search.js';
+
+import type {FloatingMarkerSizeT} from '../types.js';
+
+export default function Scenario() {
+  const [label, setLabel] = React.useState('Uber HQ');
+  const [startEnhancer, setStartEnhancer] = React.useState(true);
+  const [endEnhancer, setEndEnhancer] = React.useState(false);
+  const [floatingAnchorPosition, setFloatingAnchorPosition] = React.useState([
+    {id: 'top-left', label: 'top-left'},
+  ]);
+
+  const markers = Object.values(FLOATING_MARKER_SIZES).map(
```

Comment:
Can you remove the select and render all of the positions at the same time? This will display them all in the snapshot tests

Reply:
I will do that. The reason I had the select was so I could test the transition of the anchor position

Reply:
Ah I see. Maybe create an additional scenario?